ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1342 @@
+{
```

Comment:
<p>Can liked genres change over time? If so, maybe change to "...feature store returns <code>age=54</code> along with alice's liked genres at the time in the training data because that is the..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Correct, all other feature values can all be time-variant. </p><p> </p><p>Appended this to explicitly speak it out.</p>